packages/vsphere-cpi/charts/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Do we need to define a minimum k8s version ? Is it not compatible with <1.20 ?

Reply:
Upstream's docs say they are compatible with kubernetes v1.14+

Reply:
Below it has `Kubernetes v1.14+`. Not sure where that came from but if true then we are ok.